modules/social_features/social_follow_content/src/Plugin/ActivityContext/FollowContentActivityContext.php

```diff
@@ -39,20 +39,21 @@ public function getRecipients(array $data, $last_uid, $limit) {
   public function getRecipientsWhoFollowContent(array $related_entity, array $data) {
     $recipients = [];
 
-    $storage = \Drupal::entityTypeManager()->getStorage('flagging');
-    $flaggings = $storage->loadByProperties([
-      'flag_id' => 'follow_content',
-      'entity_type' => $related_entity['target_type'],
-      'entity_id' => $related_entity['target_id'],
-    ]);
+    $storage = $this->entityTypeManager->getStorage('flagging');
+
+    $flaggings = $storage->getQuery()
+      ->condition('flag_id', ['follow_content', 'follow_post'], 'IN')
```

Comment:
This entire change to using the query conditions isn't necessary. Looking at the `loadByProperties` function you can just pass `'flag_id' => ['follow_content', 'follow_post']` there and its underlying query mechanism will automatically use `IN` (it already does this for single values by default: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21EntityStorageBase.php/function/EntityStorageBase%3A%3AbuildPropertyQuery/8.2.x )

Doing it simply using the load method allows us to better use the framework, taking advantage of caching, query altering and possible future improvements.